University-advisor/Geocoding.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Device.Location;
+using System.Globalization;
+using University_advisor.Models;
+using Newtonsoft.Json;
+
+namespace University_advisor
+{
+    class Geocoding
+    {
+        string apiKey = "5e66cc9d64db23";
```

Comment:
In this class you were declaring your variables as var, why did you make an exception for this line?